20-kustomize/01-standalone/pingintelligence/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Kustomization
+apiVersion: kustomize.config.k8s.io/v1beta1
+# This server profile used as basis for ping-devop kubernetes info
```

Comment:
Typo in devops, also this doesn't seem applicable since this isn't a server profile?